tests/e2e/e2e_suite_test.go

```diff
@@ -38,6 +38,7 @@ func init() {
 	flag.StringVar(&oc_cli, "oc_cli", "oc", "OC CLI Client")
 	flag.StringVar(&stream, "stream", "up", "[up, down] upstream or downstream")
 	flag.Int64Var(&timeoutMultiplierInput, "timeout_multiplier", 1, "Customize timeout multiplier from default (1)")
+	flag.BoolVar(&virtTestingEnabled, "test_virt", false, "Flag to enable or disable virtualization testing, true or false")
```

Comment:
would put the same as in docs: Enable Virtual Machine backup/restore testing (default false)